### PR TITLE
New package: x64dbg.x64dbg version snapshot_2023-10-05_13-38

### LINKED ISSUE #122919: [Package Request]: x64dbg
### How can we help?

I would like someone else to build the manifest.

### Please read and ensure the following

- [X] The installer meets the above requirements

### Please provide the following information

Download Page Url: https://[x64dbg](https://x64dbg.com/).com/
Publisher: x64dbg
Package Name: x64dbg
Description: An open-source x64/x32 debugger for windows.
Package Version: 2023-10-05_13-38
Installer URL: https://sourceforge.net/projects/x64dbg/files/latest/download

### PR DESCRIPTION
**Pull request by Dragon1573**
## Description

- Resolve https://github.com/microsoft/winget-pkgs/issues/122919

## Checks

- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/122930)

### CODE REVIEW

**Comment by Dragon1573** [comment]
During test, it seems that `x64dbg.exe`, `x32dbg.exe` and `x96dbg.exe` are unable to launch with command in terminal. Go to `%LOCALAPPDATA%\Microsoft\WinGet\Links` and double-click the `.symlink` icon can launch the GUI of target application.

File: manifests/x/x64dbg/x64dbg/snapshot_2023-10-05_13-38/x64dbg.x64dbg.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Created using wingetcreate 1.5.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: x64dbg.x64dbg
+PackageVersion: snapshot_2023-10-05_13-38
```
Comment (Trenly) [inline manifests/x/x64dbg/x64dbg/snapshot_2023-10-05_13-38/x64dbg.x64dbg.installer.yaml:5]:
```suggestion
PackageVersion: 2023.10.05
```
The version should be either a semantic version or a date in order for Winget to sort the newest releases properly

File: manifests/x/x64dbg/x64dbg/snapshot_2023-10-05_13-38/x64dbg.x64dbg.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# Created using wingetcreate 1.5.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: x64dbg.x64dbg
+PackageVersion: snapshot_2023-10-05_13-38
```
Comment (Trenly) [inline manifests/x/x64dbg/x64dbg/snapshot_2023-10-05_13-38/x64dbg.x64dbg.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 2023.10.05
```

File: manifests/x/x64dbg/x64dbg/snapshot_2023-10-05_13-38/x64dbg.x64dbg.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.5.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.5.0.schema.json
+
+PackageIdentifier: x64dbg.x64dbg
+PackageVersion: snapshot_2023-10-05_13-38
```
Comment (Trenly) [inline manifests/x/x64dbg/x64dbg/snapshot_2023-10-05_13-38/x64dbg.x64dbg.yaml:5]:
```suggestion
PackageVersion: 2023.10.05
```

File: manifests/x/x64dbg/x64dbg/snapshot_2023-10-05_13-38/x64dbg.x64dbg.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+PublisherUrl: https://github.com/x64dbg
+Author: x64dbg
+PackageName: x64dbg
+PackageUrl: https://github.com/x64dbg/x64dbg
+License: GPL-3.0 License
+LicenseUrl: https://github.com/x64dbg/x64dbg/blob/development/LICENSE
+ShortDescription: An open-source x64/x32 debugger for windows.
+Moniker: x64dbg
+ReleaseNotesUrl: https://github.com/x64dbg/x64dbg/releases/tag/snapshot
+InstallationNotes: |
```
Comment (Trenly) [inline manifests/x/x64dbg/x64dbg/snapshot_2023-10-05_13-38/x64dbg.x64dbg.locale.en-US.yaml:17]:
```suggestion
InstallationNotes: |-
```

**Review by Trenly — CHANGES_REQUESTED** [review]

**Review by stephengillie — APPROVED** [review]

**Pull request merged**